.travis.yml

```diff
@@ -1,7 +1,47 @@
 language: php
+
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache
+
 php:
-    - 5.5
     - 5.6
     - 7.0
-install: composer install --no-interaction
-script: ./vendor/bin/phpunit
+    - 7.1
+
+env:
+    global:
+        - MOODLE_BRANCH=MOODLE_32_STABLE
+    matrix:
+        - DB=pgsql
+        - DB=mysqli
+
+before_install:
+    - export ORIG="$(pwd)"
+    - phpenv config-rm xdebug.ini
+    - nvm install node
+    - cd ../..
+    - composer selfupdate
+    - composer create-project -n --no-dev --prefer-dist moodlerooms/moodle-plugin-ci ci ^2
+    - export PATH="$(cd ci/bin; pwd):$(cd ci/vendor/bin; pwd):$PATH"
+
+install:
+    - moodle-plugin-ci install
+
+script:
+#   - moodle-plugin-ci phplint
```

Comment:
Which? Specifically `moodle-plugin-ci phpcpd` and `moodle-plugin-ci phpmd`? I suppose, in the first instance, I wanted to get a smaller subset of the available Moodlerooms stuff passing - in particular `moodle-plugin-ci codechecker` so the code lints to Moodle's own standard.

At a later date, once the Travis build is passing against the Moodlerooms plugins that are currently commented-in, some of the additional Moodlerooms plugins could also be commented in, and any issues then fixed. I think the `phpcpd` (copy/paste detector) and `phpmd` (mess detector) are probably very useful checks to include, in the long run, but as there's going to be a tonne of `codechecker` failures to fix first, I suppose it's arguably better to do each plugin at a time?